docs/integration/express.js.md

```diff
@@ -0,0 +1,376 @@
+title: Express.js Monitoring Integration
+description: Sematext Express.js monitoring integration is available as an npm package and required as any other Node.js module. Request rate and event loop latency, memory, http server stats, garbage collection and other Node.js reports and dashboards are available out of the box. Including HTTP logs and Application logs for API-specific troubleshooting and debugging.
+
+This is the Express.js monitoring and logging agent for Sematext, written entirely in Node.js without CPU and memory overhead. It's easy to install and require in your source code.
+
+## Sematext Express.js Agent Quick Start
+This lightweight, open-source [Express.js monitoring agent](https://github.com/sematext/sematext-agent-express) collects Node.js process and performance metrics and sends them to Sematext. It is available as an [NPM package](https://nodei.co/npm/sematext-agent-express/) that can be added to Node.js source code like any other NPM module.
+
+First you install the npm module.
+```bash
+# Terminal
+npm i sematext-agent-express
+```
+
+### Configure Environment
+
+Make sure to configure your ENVIRONMENT variables before adding `sematext-agent-express`. You can do this either by exporting the variables to your environment or by using `dotenv`. We suggest you use [dotenv](https://github.com/motdotla/dotenv).
+
+#### Export env vars
+
+If you are using the US region of Sematext Cloud:
+
+```bash
+export SPM_TOKEN=<YOUR_MONITORING_TOKEN>
```

Comment:
Maybe better to add export REGION=US above because we didn't say that US is default and that is why we are skipping it.

Reply:
Fixed.